packages/bun-types/redis.d.ts

```
@@ -630,6 +630,115 @@ declare module "bun" {
+     * @param listener The listener to call when a message is received on any
+     *                 of the subscribed channels. The listener will receive
+     *                 the message as the first argument and the channel as the
+     *                 second argument.
+     *
+     * @note Subscribing disables automatic pipeling, so all commands will be
+     *       received immediately.
+     * @note Subscribing moves the channels to a dedicated subscription state
+     *       in which only a limited set of commands can be executed.
+     */
+    subscribe(channels: string[],
+              listener: (string, string) => void): Promise<void>;
```

Biome (2.1.2)

[error] Duplicate parameter name.

The parameter overrides a preceding parameter by using the same name.

(lint/suspicious/noDuplicateParameters)